src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,124 @@
+* **User**: In your organization, your [New Relic users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/) are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. 
+
+## Single account structure [#single-account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these apply to you, then you might only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.
+
+## Create multiple accounts [#multiple-accounts]
+
+There are many reasons to create multiple accounts within your organization. Here are some common examples that can help you decide if a multi-account structure is right for you and your team.
+
+### Use multiple accounts to separate testing and production environments [#environments]
+
+Multiple accounts let you observe data across multiple environments so you can monitor changes before they go to production. If you’re updating your application regularly, it's important that you have a strong testing environment in place so you can make sure each deployment is error free.
+
+<img
+    style={{ align: 'left',maxWidth: '100%' }}
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Separate projects with unique data sets with multiple accounts [#data-sets]
+
+If you manage a payment provider that works in multiple countries, you might need to manage different currencies and different tax regulations. Or, you could manage a social media site and you might need to comply with different data collection policies in different regions. In both those situations, it is a good idea to create multiple accounts. 
+
+<img
```

Comment:
This image might be a bit clearer if you represented the organization as a box (container) that holds the three accounts. In this diagram, it almost looks like the accounts are not contained within the organization.

Reply:
I think I fixed this language too. Good callout, Rob.